pjsip/include/pjsip/sip_transport.h

```diff
@@ -245,6 +265,7 @@ typedef struct pjsip_tpselector
         pjsip_transport *transport;
         pjsip_tpfactory *listener;
         void            *ptr;
+        pjsip_tpselector_ip_ver ip_ver;
```

Comment:
minor: `ptr` seems to be the least important field in the union (for generic/unclear purpose :D ), so perhaps move it to the last?